pkg/innerring/processors/netmap/nodevalidation/external/calls.go

```diff
@@ -0,0 +1,153 @@
+package external
+
+import (
+	"bytes"
+	"crypto/rand"
+	"encoding/binary"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
+	neofsecdsa "github.com/nspcc-dev/neofs-sdk-go/crypto/ecdsa"
+	"github.com/nspcc-dev/neofs-sdk-go/netmap"
+)
+
+// Signature represents a digital signature of a signed message.
+type Signature struct {
+	Sign []byte `json:"sign"`
+}
+
+// SignedRequest represents a request containing node information
+// and its corresponding digital signature.
+type SignedRequest struct {
+	NodeInfo  json.RawMessage `json:"node_info"`
+	Nonce     uint32          `json:"nonce"`
+	Signature *Signature      `json:"signature"`
+}
+
+// Body represents the body of the response from an external validator.
+type Body struct {
+	Verified bool   `json:"verified"`
+	Details  string `json:"details"`
+}
+
+// SignedResponse represents the result of a verification
+// process from an external validator.
+// Verified indicates whether the verification was successful.
+type SignedResponse struct {
+	Result    json.RawMessage `json:"result"`
```

Comment:
And nonce should be a part of the result, the same nonce as the one sent to the validator.